.github/workflows/bundle_size.yml

```diff
@@ -9,10 +9,10 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
-      - name: Install Node.js 18
+      - name: Install Node.js 20
         uses: actions/setup-node@v4
         with:
-          node-version: 18
+          node-version: 20
```

Comment:
just for consideration, pull the version up to an env for the job, and probably don't need to set the step name to the version, seems like that requires superfluous maintenance?:

```
jobs:
  build:
    env:
      NODE: "XX"
...

    steps:
      - name: Install Node
         with:
           node-version: ${{env.NODE}}

```

Reply:
Great idea! Done in 0d85200e.